framework/src/transfers/MultiAppConservativeTransfer.C

```diff
@@ -51,7 +52,9 @@ MultiAppConservativeTransfer::MultiAppConservativeTransfer(const InputParameters
     _to_var_names(getParam<std::vector<AuxVariableName>>("variable")),
     _preserve_transfer(isParamValid("from_postprocessors_to_be_preserved")),
     _from_postprocessors_to_be_preserved(
-        getParam<std::vector<PostprocessorName>>("from_postprocessors_to_be_preserved")),
+        _preserve_transfer
+            ? getParam<std::vector<PostprocessorName>>("from_postprocessors_to_be_preserved")
```

Comment:
use a default instead

Reply:
Default value can not be used here. See the line above for `_preserve_transfer `. If default value is used, `_preserve_transfer` will always be true.

Reply:
ok that works